services/cypress/integration/adminWorkflow/settings.spec.ts

```diff
@@ -1,63 +1,148 @@
-import {interceptors} from 'axios';
-
 describe('Admin user can view application level settings', () => {
     beforeEach(() => {
         cy.stubFeatureFlags()
         cy.interceptGraphQL()
     })
 
-    it('and update user division assignments', () => {
-        // make sure a cms user in db first
-        cy.logInAsCMSUser()
+    // it('and update user division assignments', () => {
```

Comment:
I think we can delete this commented out test, seems like the new test addresses both